### PR TITLE
Run matrix-react-sdk tests on merge queue

### PR DESCRIPTION
**Pull request by t3chguy**
Requires https://github.com/matrix-org/matrix-react-sdk/pull/10174
Requires https://github.com/matrix-org/matrix-react-sdk/pull/10198

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -59,3 +61,21 @@ jobs:
                   path: |
                       coverage
                       !coverage/lcov-report
+
+    matrix-react-sdk:
+        name: Downstream test matrix-react-sdk
+        if: github.event_name == 'merge_group'
+        uses: matrix-org/matrix-react-sdk/.github/workflows/tests.yml@develop
```
Comment (gsouquet) [inline .github/workflows/tests.yml:68]: Could we run this workflow without any of the coverage option? 
That will significantly speed up test execution, and we don't need to check for coverage in this scenario

**Review by gsouquet — CHANGES_REQUESTED** [review]
Otherwise looks good to me 👍

File: .github/workflows/tests.yml (hunk 1 above)
Comment (t3chguy) [reply .github/workflows/tests.yml:68]: Done, requires https://github.com/matrix-org/matrix-react-sdk/pull/10198

**Review by gsouquet — APPROVED** [review]
lgtm!

**Pull request merged**